docs/source/getting-started/setting-up-hypervisor-driver-plugin.adoc

```diff
@@ -75,14 +74,14 @@ $ newgrp libvirt
 ----
 
 [[xhyve-driver-install]]
-== xhyve driver
+=== xhyve driver
```

Comment:
Double equal to sign only to make it visible as part of table of content. Click on `View` button of this page.